grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -65,7 +65,9 @@
 public final class GrpcServiceBuilder {
 
     private static final Set<SerializationFormat> DEFAULT_SUPPORTED_SERIALIZATION_FORMATS =
-            ImmutableSet.of(GrpcSerializationFormats.PROTO, GrpcSerializationFormats.PROTO_WEB);
+            ImmutableSet.of(GrpcSerializationFormats.PROTO,
+                            GrpcSerializationFormats.PROTO_WEB,
+                            GrpcSerializationFormats.PROTO_WEB_TEXT);
```

Comment:
Could you enable all gRPC protocols by default?
Most of Armeria users use gRPC-Java and `com.google.protobuf.Message`. Armeria can encode and decode the messages by default.

Even though Armeria does not support ScalaPB JSON by default. 😅
Any thoughts? @anuraaga

Reply:
Oops... Ditto 🤣